pkg/generator/generator.go

```diff
@@ -352,10 +364,20 @@ func renderBuildFiles(buildDir, repoPath, projectName string) error {
 	dTd := tmplData{
 		ProjectName: projectName,
 	}
-	if err := renderFile(buf, "tmp/build/Dockerfile", dockerFileTmpl, dTd); err != nil {
+
+	if err := renderWriteFile(filepath.Join(buildDir, dockerfile), "tmp/build/Dockerfile", dockerFileTmpl, dTd); err != nil {
+		return err
+	}
+
+	if err := renderWriteFile(filepath.Join(buildDir, "test-framework", testingDockerfile), "tmp/build/test-framework/Dockerfile", testingDockerFileTmpl, dTd); err != nil {
+		return err
+	}
+
+	buf = &bytes.Buffer{}
+	if err := renderFile(buf, filepath.Join(buildDir, goTest), goTestScript, tmplData{}); err != nil {
 		return err
 	}
-	return renderWriteFile(filepath.Join(buildDir, dockerfile), "tmp/build/Dockerfile", dockerFileTmpl, dTd)
+	return writeFileAndPrint(filepath.Join(buildDir, goTest), buf.Bytes(), os.FileMode(int(0755)))
```

Comment:
Use the constant `defaultExecFileMode `.

Reply:
defaultExecMode is currently set to 0744, which doesn't work for this setup. Should I change defaultExecMode to 0755?